FindCEF.cmake

```diff
@@ -1,91 +1,155 @@
 include(FindPackageHandleStandardArgs)
 
-SET(CEF_ROOT_DIR "" CACHE PATH "Path to a CEF distributed build")
-
-message(STATUS "Looking for Chromium Embedded Framework in ${CEF_ROOT_DIR}")
-
-find_path(CEF_INCLUDE_DIR "include/cef_version.h"
-	HINTS ${CEF_ROOT_DIR})
-
-if(APPLE)
-	find_library(CEF_LIBRARY
-		NAMES cef libcef cef.lib libcef.o "Chromium Embedded Framework"
-		NO_DEFAULT_PATH
-		PATHS ${CEF_ROOT_DIR} ${CEF_ROOT_DIR}/Release)
-	find_library(CEFWRAPPER_LIBRARY
-		NAMES cef_dll_wrapper libcef_dll_wrapper
-		NO_DEFAULT_PATH
-		PATHS ${CEF_ROOT_DIR}/build/libcef_dll/Release
-			${CEF_ROOT_DIR}/build/libcef_dll_wrapper/Release
-			${CEF_ROOT_DIR}/build/libcef_dll
-			${CEF_ROOT_DIR}/build/libcef_dll_wrapper)
-elseif(UNIX)
-	find_library(CEF_LIBRARY
-		NAMES libcef.so "Chromium Embedded Framework"
-		NO_DEFAULT_PATH
-		PATHS ${CEF_ROOT_DIR} ${CEF_ROOT_DIR}/Release)
-	find_library(CEFWRAPPER_LIBRARY
-		NAMES libcef_dll_wrapper.a
-		NO_DEFAULT_PATH
-		PATHS ${CEF_ROOT_DIR}/build/libcef_dll_wrapper
-			${CEF_ROOT_DIR}/libcef_dll_wrapper)
+set(CEF_ROOT_DIR
+    ""
+    CACHE PATH "Path to a CEF distributed build")
+mark_as_advanced(CEF_ROOT_DIR)
```

Comment:
I would vote that because we consider obs-browser to be recommended for core building, this shouldn't be marked as Advanced so that users can more clearly see it pre-populated in CMake-GUI.

Reply:
Sounds good, changed it.